build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -27,8 +27,7 @@ const {app} = require('../../server/test-server');
 const {createKarmaServer, getAdTypes} = require('./helpers');
 const {getFilesFromArgv} = require('../../common/utils');
 const {green, yellow, cyan, red} = require('ansi-colors');
-const {isCiBuild} = require('../../common/ci');
-const {isGithubActionsBuild} = require('../../common/github-actions');
+const {isCiBuild, isGithubActionsBuild} = require('../../common/ci');
```

Comment:
This file should be CI-agnostic too, right?

Reply:
The non-agnostic part is in deciding which set of tests to run (all tests vs. cross-browser only).

https://github.com/ampproject/amphtml/blob/a1fa9e4fb476b9a922f6e0e8a6abcd12b7d96ea5/build-system/tasks/runtime-test/runtime-test-base.js#L131-L133